src/cannon-es-debugger.ts

```diff
@@ -30,13 +30,12 @@ export type DebugOptions = {
   scale?: number
   onInit?: (body: Body, mesh: Mesh, shape: Shape) => void
   onUpdate?: (body: Body, mesh: Mesh, shape: Shape) => void
-  autoUpdate?: Boolean
 }
 
 export default function cannonDebugger(
```

Comment:
This does not match what is in the README:

```ts
export default class CannonDebugger {
  constructor(scene: Scene, world: World, options: DebugOptions): void
  update(): void
}
```

Reply:
we're using this pattern because we don't want to expose the private variables, we can switch to a class when we can ship private class fields